src/systems/sprites/sprite.vert

```diff
@@ -0,0 +1,28 @@
+#version 300 es
+
+uniform mat4 modelViewMatrix;
+uniform mat4 projectionMatrix;
+
+in vec4 mvCol0;
```

Comment:
Matrix transforms per vertex is a bit too heavy. I'd store matrix transforms in a uniform array and store an index into that array per vertex.

Reply:
Better yet since all of these are all always just flat quads, we can send way less data by using instancing, and sending these transforms as instance attributes.

Reply:
Leaving instancing / reducing per-vertex data as a TODO for now, especially given that we will likely work to combine our "batching" efforts in the near future.